src/addons/i2cdisplay.cpp

```diff
@@ -48,6 +48,7 @@ void I2CDisplayAddon::process() {
 		drawStatusBar(gamepad);
 		drawText(0, 3, "[Web Config Mode]");
 		drawText(0, 4, std::string("GP2040-CE : ") + std::string(GP2040VERSION));
+		drawText(0,5, "[http://192.168.7.1]");
```

Comment:
Add spacing between parameters to keep format consistent.

drawText(0, 5, "[http://192.168.7.1]");